Resources/translations/system-configuration.en.xliff

```diff
@@ -14,6 +14,10 @@
                 <source>label.easy_backup.setting_backup_dir</source>
                 <target>Backup directory</target>
             </trans-unit>
+            <trans-unit id="label.easy_backup.setting_paths_to_be_backuped">
+                <source>label.easy_backup.setting_paths_to_be_backuped</source>
+                <target>Files and paths to be backed up</target>
```

Comment:
https://www.deepl.com/translator#de/en/Zu%20sichernde%20Dateien%20und%20Pfade
```suggestion
                <target>Files and paths to backup</target>
```

Reply:
I changed it in the code as well. Thanks